### PR TITLE
Add keyword SALINITY with unit molality

### PR DESCRIPTION
**Pull request by totto82**
I need something to set the constant salinity used by the CO2STOR model. 
See OPM/opm-material#427

### CODE REVIEW

File: opm/parser/eclipse/Units/Units.hpp [hunk 1]
```diff
@@ -274,6 +275,7 @@ namespace Opm {
         constexpr const double FoamDensity          = kilogram/cubic(meter);
         constexpr const double FoamSurfactantConcentration = kilogram/cubic(meter);
         constexpr const double Salinity             = kilogram/cubic(meter);
+        constexpr const double Molality             = mol/kilogram; //1e-3 kg-M/kg
```
Comment (joakim-hove) [inline opm/parser/eclipse/Units/Units.hpp:278]: There seems to be missing a `kilogram` in the numerator?

**Comment by joakim-hove** [comment]
Seems you have made quite some changes to introduce a dimensionless unit - does that make sense?

**Comment by totto82** [comment]
> Seems you have made quite some changes to introduce a dimensionless unit - does that make sense?

Probably not. Lets keep it simple.

**Comment by joakim-hove** [comment]
Cool - I'll kick Jenkins and then she goes in!

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**